### PR TITLE
USWDS-Site - Accordion: Add accordion color settings

### PR DESCRIPTION
**Pull request by mejiaj**
# Summary

Documenting new accordion color settings & updating changelog entries.

## Related issue

Part of  uswds/uswds#5269.

## Preview link

Preview link:
- [Accordion settings](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/jm-accordion-background/components/accordion/#using-the-accordion-component-2)
- [Accordion changelog](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/jm-accordion-background/components/accordion/#latest-updates)
- [Settings page entries](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/jm-accordion-background/documentation/settings/#accordion-2)
- [Settings changelog](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/jm-accordion-background/documentation/settings/#latest-updates)

## Problem statement

Settings and changelog entries need to be updated for accuracy.

## Solution

1. Add entries to `_data/settings/components/accordion.yml`
2. Update changelog for both settings & component

## Testing and review

**Accordion component**
1. Visit [Accordion](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/jm-accordion-background/components/accordion) component
2. Settings table should include two new background color settings
3. There should be a new changelog entry

**Settings page**
1. Visit [Settings page](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/jm-accordion-background/documentation/settings/#accordion-2)
2. New accordion settings should be documented
3. Go to changelog section at the bottom of the page
4. There should be a new changelog entry for new settings

### CODE REVIEW

File: _data/settings/components/accordion.yml [hunk 1]
```diff
@@ -1,5 +1,10 @@
 name: Accordion
 contents:
+  - name: Background color
+    description: Content background color of the accordion.
```
Comment (amyleadem) [inline _data/settings/components/accordion.yml:4]: A possible tiny tweak:
```suggestion
    description: Background color of the accordion content.
```

File: _data/settings/components/accordion.yml [hunk 2]
```diff
@@ -10,6 +15,11 @@ contents:
     var: $theme-accordion-border-color
     default: '"base-lightest"'
     type: color
+  - name: Button background color
+    description: Button background color of the accordion.
```
Comment (amyleadem) [inline _data/settings/components/accordion.yml:19]: A possible tiny tweak:
```suggestion
    description: Background color of the accordion button.
```

**Review by amyleadem — CHANGES_REQUESTED** [review]
Looking good, @mejiaj ! I added comments for a couple possible changes. Let me know if you have questions:

Checks performed:

- [x] Confirmed the updated settings match the updates from https://github.com/uswds/uswds/pull/5269/
- [x] Checked for grammar and spelling

**Review by mahoneycm — APPROVED** [review]
Looking good on my end!

I agree with @amyleadem comments to add the setting names to the change log as well as the adjustments to the setting's descriptions for improved clarity. 

No further adjustments!

- [x] Checked for changelog items on Accordion and Settings page
- [x] Checked for setting's descriptions on Accordion page
- [x] Checked for new accordion settings on Settings page

File: _data/changelogs/docs-settings.yml [hunk 3]
```diff
@@ -2,6 +2,17 @@ title: Settings
 type: utility
 changelogURL:
 items:
+  - date: 2023-05-03
+    summary: Added accordion color settings.
+    summaryAdditional:
```
[the remark this answers is not in the document]
Comment (mejiaj) [reply _data/changelogs/docs-settings.yml]: Good idea, added in a72a5fd.

File: _data/changelogs/component-accordion.yml [hunk 4]
```diff
@@ -2,6 +2,16 @@ title: Accordion
 type: component
 changelogURL:
 items:
+  - date: 2023-05-03
+    summary: Added two new settings to customize accordion background colors.
+    summaryAdditional: You can now customize accordion button and content background colors with `$theme-accordion-button-background-color` and `$theme-accordion-background-color`.
+    isBreaking: false
+    affectsJavascript: false
+    affectsMarkup: false
+    affectsStyles: true
```
Comment (amyleadem) [inline _data/changelogs/component-accordion.yml:11]: One last thing - I just noticed that we now have an `affectsSettings` option. What do you think about applying it to both changelog entries here?

**Review by amyleadem — CHANGES_REQUESTED** [review]
Looks good! I just had one question related to `affectsSettings`.

**Comment by amyleadem** [comment]
One more note: thinking it would be a good idea to also add a changelog for the checkbox component since the calculation for determining the check icon color has been fixed with this update as well.

File: _data/changelogs/component-accordion.yml (hunk 4 above)
Comment (mejiaj) [reply _data/changelogs/component-accordion.yml:11]: Added it in df8fb4b.

File: _data/changelogs/docs-settings.yml [hunk 5]
```diff
@@ -2,6 +2,18 @@ title: Settings
 type: utility
 changelogURL:
 items:
+  - date: 2023-05-03
```
Comment (amyleadem) [inline _data/changelogs/docs-settings.yml:5]:
```suggestion
  - date: NNNN-NN-NN
```

File: _data/changelogs/component-accordion.yml [hunk 6]
```diff
@@ -2,6 +2,17 @@ title: Accordion
 type: component
 changelogURL:
 items:
+  - date: 2023-05-03
```
Comment (amyleadem) [inline _data/changelogs/component-accordion.yml:5]:
```suggestion
  - date: NNNN-NN-NN
```

File: _data/changelogs/component-accordion.yml [hunk 7]
```diff
@@ -2,6 +2,15 @@ title: Accordion
...
 items:
+  - date: NNNN-NN-NN
+    summary: Added two new settings to customize accordion background colors.
+    summaryAdditional: You can now customize accordion button and content background colors with `$theme-accordion-button-background-color` and `$theme-accordion-background-color`.
+    isBreaking: false
+    affectsJavascript: false
+    affectsMarkup: false
+    affectsStyles: true
+    affectsSettings: true
+    githubPr: 5269
```
Comment (amyleadem) [inline _data/changelogs/component-accordion.yml:13]:
```suggestion
    githubPr: 5269
    githubRepo: uswds
```

**Review by amyleadem — APPROVED** [review]

**Review by thisisdano — APPROVED** [review]

**Pull request merged**